README.md

```diff
@@ -104,6 +104,30 @@ Note: The faucet is limited to one transfer per wallet every 6500 blocks (~1 day
 1. [avax faucet](https://faucet.avax-test.network/)
 2. [explorer](https://explorer.avax-test.network/)
 
+## Gitpod Continuous Dev Environment
+
+This repo is configured to work with Gitpod.
+
+### New Contributors
+
+If you are a new contributor, you can fork the repo and start a pre-configured gitpod environment by prefixing your fork URL with `gitpod.io/#`. For example:
+
+`https://gitpod.io/#https://github.com/.../...`
+
+Then follow the standard [Github fork & PR workflow](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+### Permissioned Contributors
+
+If you are an established contributor with access rights to create and push to branches in this repo, you can use a simpler flow.
+
+1. Obtain a Personal Access Token from your github UI.
+2. In your gitpod dashboard, set a new variable named `GITHUB_OHM_PERSONAL_ACCESS_TOKEN` to the value of the access token.
+3. Use the button below to start a pre-configured gidpod environment.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/OlympusDAO/olympus-frontend)
+
+4. Follow the simplified [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow) to create new branches in the repo and submit PRs.
+
```

Comment:
can you mark your gitpod steps as optional? I don't want new contributors to think that's required or recommended.